packages/flare/python/cron_job_ingest_events.py

```diff
@@ -73,6 +73,12 @@ def main() -> None:
 
         next = get_next(app=app, tenant_id=tenant_id)
         start_date = get_start_date(app=app)
+        logger.debug(
```

Comment:
I see a lot of `.format()` in your code. Can you remove them and instead use f-strings?

The advantage with f-strings is that mypy can validate them.

Reply:
Didn't know about that one, it's way easier to read too